openstack/antiddos/v2/alarmreminding/requests.go

```diff
@@ -4,10 +4,55 @@ import (
 	"github.com/chnsz/golangsdk"
 )
 
-func WarnAlert(client *golangsdk.ServiceClient) (r WarnAlertResult) {
-	url := WarnAlertURL(client)
+func GetWarnAlert(client *golangsdk.ServiceClient) (r WarnAlertResult) {
+	url := GetWarnAlertURL(client)
 	_, r.Err = client.Get(url, &r.Body, &golangsdk.RequestOpts{
 		OkCodes: []int{200},
```

Comment:
When the correct corresponding status code is 200, there is no need to specially specify `OkCodes`.